gnomenews/view.py

```diff
@@ -189,27 +189,43 @@ def __init__(self, tracker, post):
               h1 {
                 margin-bottom: 2px;
               }
+
+              #footer {
+                font-size: 14px;
+                color: #777;
+              }
+
+              #footer a:link,
+              #footer a:active,
+              #footer a:visited {
+                text-decoration: none;
+                color: #777;
+              }
+
+              #footer a:hover {
+                text-decoration: underline;
+              }
             </style>
             <body>
               <article>
               <h1>%s</h1>
-              <span>
-            """ % post.title
+              <span>%s</span>
+              <p>%s</p>
+              <div id="footer">""" % (post.title, post.author, post.content)
 
             if post.author_homepage:
-                html += """<a href="%s">%s</a>""" % (post.author_homepage, post.author)
-            else:
-                html += """%s""" % post.author
-
-            html += """<p>%s</p>""" % post.content
+                html += """<p>%s</p>""" % post.author_homepage
 
             if post.author_email:
-                html += """<small>%s</small>""" % post.author_email
+                html += """<p>%s</p>""" % post.author_email
```

Comment:
I think it would be best to combine author name with email and convert it into a link.

Not sure if having a post link below the post is a good idea - I'd put all the metadata right after title